tokio-util/tests/compat.rs

```diff
@@ -0,0 +1,42 @@
+#![cfg(all(feature = "compat", not(tokio_wasi)))] // WASI does not support all fs operations
```

Comment:
The `tokio_wasi` marker only works in the main Tokio crate. (via the build script)